maquetado/scss/_styles.scss

```
@@ -1,3 +1,7 @@
 * {
   font-family: $font-family;
 }
+
+.wrapper {
+  padding: $nav-height + 50px 30px 30px;
```

```suggestion
  padding: #{$nav-height + 50px} 30px 30px;
```
This is because in some versions of SASS doesn't work like that

---

I didn't know that consideration. I'm gonna read about that.